src/init.cpp

```diff
@@ -1414,6 +1414,8 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
                 // LoadBlockIndex will load fTxIndex from the db, or set it if
                 // we're reindexing. It will also load fHavePruned if we've
                 // ever removed a block file from disk.
+                // Note that it also sets fReindex based on the disk flag!
+                // From here on out fReindex and fReset mean something different!
```

Comment:
In commit "Fix resume-of-reindex-after-restart"

Maybe make this more specific. You could say "From here on out fReindex may be true even if fReset is false (indicating the node was shut down mid-reindex)."